README.md

```diff
@@ -55,6 +53,22 @@ or, to combine all three, simply run `./manage.py precommit`.
 
 You can also set up `pylint`, `isort`, `black` and `prettier` in your IDE to avoid doing this manually all the time.
 
+### Creating a Pull Request (Workflow Suggestion)
+1. (once) [Fork](https://github.com/e-valuation/EvaP/fork) the repository so you have a GitHub Repo that you have write access to.
+
+2. (once) Set up some Authentication for GitHub that allows push access. A common option is using [SSH keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/about-ssh), the remaining instructions assume an SSH key setup. An alternative is using the [GitHub CLI tool](https://cli.github.com/).
+
+3. (once) Ensure your [git remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) are setup to use SSH. To fetch the up-to-date state of the official repo, it's always useful to have an "upstream" remote configured:
+   ```bash
+   git remote set-url origin git@github.com:<your-username>/EvaP.git
+   git remote add upstream git@github.com:e-valuation/EvaP.git
```

Comment:
Huh, when did they change that `set-url` actually sets both fetch and push urls? I thought I remembered that it was odd that you always had to run it once without and once with `--push` to actually set both urls, but this here works also for me :D

I wonder if we should maybe provide a more idempotent set of commands here:
```bash
git remote remove origin
git remote add origin git@github.com:<your-username>/EvaP.git
git remote add upstream git@github.com:e-valuation/EvaP.git
```

Reply:
> Huh, when did they change that set-url actually sets both fetch and push urls? I thought I remembered that it was odd that you always had to run it once without and once with --push to actually set both urls, but this here works also for me :D

Fair point. The way I understand the documentation, `set-url` without any further arguments will simply change the "first" url configured for the remote, and perhaps this is the push and pull url with the trivial setup after cloning? That would mean you'd have to manually set the push url once you specified a separate push url -- that would make sense to me.

I didn't want to mess with the origin branch in any way that could mess up the upstream-branch configuration for main, that's why I wanted to keep origin around but re-point it, but I see the argument about idempotency. How does a branch "remember" its upstream branch, is it just by name, or is it messed up after removing the remote and adding a new remote with the same name?

Reply:
Removing and re-adding the remote breaks the branch upstream configuration of existing branches. `set-url` should be as idempotent as remove-add. The main disadvantage of `set-url` is if multiple URLs are configured, but we don't expect that here (otherwise the user probably knows how remotes work).

Left as `set-url` for now. We'll see if we run into any trouble with it.